draft-ietf-quic-recovery.md

```diff
@@ -1141,13 +1141,16 @@ SetLossDetectionTimer():
     // Time threshold loss detection.
     loss_detection_timer.update(earliest_loss_time)
     return
-    
-  if (IsServerWhichReachedAddressValidationLimit()):
+
+  if (server is at anti-amplification limit):
+    // The server's alarm is not set if nothing can be sent.
     loss_detection_timer.cancel()
     return
 
   if (no ack-eliciting packets in flight &&
-      PeerNotAwaitingAddressValidation()):
+      peer not awaiting address validation):
+    // There is nothing to detect lost and the server
+    // is not blocked by the anti-amplification limit.
```

Comment:
This comment is a little strange.

The point here is to catch the case where there is nothing to send, with the exception that the client might need to keep the server primed in case it is blocked on its anti-amplification limit.

As we've already dispensed with the anti-amplification limit at the server, I think that this needs to concentrate on the client instead.

```suggestion
    // There is nothing to detect lost, so set no timer.
    // However, the client still sets the timer if the server
    // might be blocked by the anti-amplification limit.
```